spinnaker_camera_driver/scripts/linux_setup_flir.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
+
+echo ""
+echo -e "\e[32mStarting spinnaker_camera_driver setup script\e[0m"
+echo ""
+
+sudo addgroup flirimaging
+sudo usermod -a -G flirimaging ${USER}
+
+val=$(< /sys/module/usbcore/parameters/usbfs_memory_mb)
+if [ "$val" -lt "1000" ]; then
+  if [ -e /etc/default/grub ]; then
+    if [ $(grep -c "usbcore.usbfs_memory_mb=1000" /etc/default/grub) -eq 0 ]; then
+      sudo sed -i 's/GRUB_CMDLINE_LINUX_DEFAULT="[^"]*/& usbcore.usbfs_memory_mb=1000/' /etc/default/grub
+      echo "Increased the usbfs memory limits in the default grub configuration. Updating grub"
+      sudo update-grub
+    else
+      echo -e "\e[33mWarn: usbfs memory limit is already set in /etc/default/grub. No changes made, try rebooting the computer\e[0m"
+    fi
+
+  else
+    echo -e "\e[33mWarn: /etc/default/grub configuration file not found, no changes made. usbfs_memory_mb must be set manually. See docs/linux_setup_flir.md for instructions\e[0m"
```

Comment:
The `docs/linux_setup_flir.md` won't exist when install via debian, a link to the repo can be used instead.

Reply:
I suggest to leave the "=1000" part out of the grep line such that you don't modify the grub line if any value has been set. Then in the warning message echo the line and ask the user to verify that it is set to >= 1000